install/kubernetes/cilium/values.yaml

```diff
@@ -129,6 +129,10 @@ global:
     # TCP port for the agent health API. This is not the port for cilium-health.
     healthPort: 9876
 
+  operator:
```

Comment:
hope you don't mind if I chime in.

Understand that this param is added in global object so that the config map can grab it. However, if I am not wrong, there is one initiative to reduce the usage of global values, and make each sub chart more independent and self-contained.

Reply:
This is correct, @mrostecki can you do something that was similarly being done in https://github.com/cilium/cilium/pull/12097?

Reply:
Sure, I will just add it to `install/kubernetes/cilium/charts/config/values.yaml`.